### PR TITLE
workflows: Consolidate, update and remove travis references

### PR DESCRIPTION
**Pull request by jrha**
Use naming and base images that are consistent with other repositories.

### CODE REVIEW

File: .github/workflows/continuous-integration.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+jobs:
+
+  test-library:
+    runs-on: ubuntu-latest
+    container: ghcr.io/quattor/quattor-test-container:latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+    - name: Install dependencies
+      run: yum -y install subversion
```
Comment (ned21) [inline .github/workflows/continuous-integration.yaml:14]: Seems a bit odd to need this? Does one of the scripts depend on it?

Comment (jrha) [reply .github/workflows/continuous-integration.yaml:14]: `create-vanilla-SCDB` which is used to test the templates does

Comment (wpoely86) [inline .github/workflows/continuous-integration.yaml:14]:
```suggestion
      run: dnf -y install subversion
```

**Review by wpoely86 — APPROVED** [review]

**Comment by wpoely86** [comment]
@jrha same here:
```
      [panc] evaluation error [/tmp/scdb-vanilla/cfg/grid/umd-3/defaults/grid/config.pan:1197.62-1208.13]
     [panc] second argument for logical-and (&&) operation must be a boolean value
     [panc] >>> call stack trace 
     [panc] >>> [/tmp/scdb-vanilla/cfg/grid/umd-3/defaults/grid/config.pan:1197.62-1208.13]
     [panc] >>> [/tmp/scdb-vanilla/cfg/grid/umd-3/machine-types/grid/base.pan:169.1-169.35]
     [panc] >>> [/tmp/scdb-vanilla/cfg/grid/umd-3/machine-types/grid/ce.pan:49.1-49.38]
     [panc] >>> [/tmp/scdb-vanilla/cfg/clusters/grid/umd3/profiles/cream.example.org.pan:13.1-13.32]
```

**Comment by jrha** [comment]
This is the same real issue from merging https://github.com/quattor/template-library-standard/pull/161.

**Pull request merged**